Create sowanicj.txt

Sowanicj opened the pull request:
Added my file, with onid name and github name

Sowanicj committed:
Update sowanicj.txt

Added my file, with onid name and github name

meynm-cs361-ta commented:
You added your file; that's good. Now all you need to do is please update the contents of the file with your name and ONID id and re-commit it. Thanks!

Sowanicj commented:
Oh, crap, thank you! Working on that now!

Sowanicj committed:
Update sowanicj.txt

Sowanicj commented:
Updated sowanicj.txt to contain ONID id and username

Aelianos committed:
Fixed sowanicj.txt

I originally thought contents were onid username and github username. Fixed it to be my name and github username
